.vscode/twenty.code-workspace

```diff
@@ -43,21 +43,21 @@
     "[typescript]": {
       "editor.formatOnSave": false,
       "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true,
+        "source.fixAll.eslint": "explicit",
```

Comment:
do we want this change?

Reply:
Don't really know but it's automatically changed on my VsCode each time